app/views/activities/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="form-container">
+  <div class="row justify-content-center">
+    <div class="col-6">
+      <h1>New Activity</h1>
+        <%= simple_form_for(@activity) do |f| %>
+        <%= f.input :title %>
+        <%= f.input :description %>
+        <%= f.input :level, :collection => %w[Beginner Intermediate Advance]%>
+        <%= f.input :location %>
+        <%= f.input :price, input_html: { min: '0', step: '1' }%>
+        <%= f.input :group_size, input_html: { min: '1', max: '25' }%>
+        <%= f.hidden_field :user_id, value: current_user.id %>
```

Comment:
Maybe this could be redundant someway. Maybe after we implement 'Pundit' but the form cannot be submitted without the user_id.
Any suggestions on better implementation or is it sufficient for now!

Reply:
Yes, let's change this if we can once we are using Pundit. If it works like this, that's ok.